client/src/Components/DualRangeSlider/DualRangeSlider.tsx

```diff
@@ -1,65 +1,64 @@
-import React, { useCallback, useEffect, useState, useRef } from 'react';
+import React, { useCallback, useEffect, useRef } from 'react';
 
-const DualRangeSlider = ({ min, minValue, max, maxValue, onChangeMin, onChangeMax }) => {
-  const [minVal, setMinVal] = useState(minValue);
-  const [maxVal, setMaxVal] = useState(maxValue);
+const DualRangeSlider = ({ min, minValue, max, maxValue, updateMagnitudeMin, updateMagnitudeMax }) => {
```

Comment:
```suggestion
const DualRangeSlider = ({ min, minValue, max, setValue, setMin, setMax }) => {
```
change this name would make the componetn a bit more generic.